packages/kbn-dom-drag-drop/src/sass/drag_drop_mixins.scss

```diff
@@ -0,0 +1,46 @@
+// from variables:
+
+$domDragDropZLevel0: 0;
+$domDragDropZLevel1: 1;
+$domDragDropZLevel2: 2;
+$domDragDropZLevel3: 3;
```

Comment:
Similar to my previous comment, would it be better to change these SCSS variable prefixes from `$domDragDrop` to `$kbnDragDrop`?